### ISSUE TITLE
Ngrok 3.x callback-style deprecated

### ISSUE THREAD

**Issue by DJTB** [react-boilerplate/react-boilerplate#2265]
- [x] Bug (https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md#bug-reports)
- [ ] Feature (https://github.com/react-boilerplate/react-boilerplate/blob/master/.github/CONTRIBUTING.md#feature-requests)

## Description

Tunneling dev appears to do nothing.

## Steps to reproduce

`npm run start:tunnel`

# Versions

- React-Boilerplate 3.6
- Node: 8.11.0

@gretzky Not sure which branch I would PR to (since dev is being cleaned up).

TL:DR; need to switch ngrok to async/await.
Quick fix for `./server/index.js`:
```
if (ngrok) {
    (async () => {
      let url;
      try {
        url = await ngrok.connect(port);
      } catch (ngrokErr) {
        logger.error(ngrokErr);
      }
      logger.appStarted(port, prettyHost, url);
    })();
  } else {
    logger.appStarted(port, prettyHost);
  }
```

**Comment by julienben** [comment]
I updated ngrok to the new syntax and tested it before the v3.6 update. Isn't the necessary async/await code already present in https://github.com/react-boilerplate/react-boilerplate/blob/master/server/index.js ?

**Comment by julienben** [comment]
In this commit: https://github.com/react-boilerplate/react-boilerplate/commit/ebef2ed570a222d126d31b833748d3410378b99f

**Comment by DJTB** [comment]
Oh hmm, I diffed/merged master with a current project and must have missed this somehow.
My bad!
